tests/unit/pages/lesson-details/LessonDetails.spec.jsx

```diff
@@ -68,6 +87,32 @@ describe('LessonDetails', () => {
     })
   })
 
+  it('should render mark button when userRole is student', async () => {
```

Comment:
Not `async` currently
```suggestion
  it('should render mark button when userRole is student', () => {
```

Reply:
done